unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/Tests/WearableGridItemComponentViewShould.cs

```diff
@@ -276,23 +275,6 @@ public void EnableHoverSelectedEquippedContainer()
             Assert.IsTrue(view.hoverSelectedEquippedContainer.activeSelf);
         }
 
-        [TestCaseSource(nameof(GetAllRarities))]
-        public void EnableRarityBackground(NftRarity nftRarity)
```

Comment:
this test should not be removed, instead rework it so it applies to the new rarity background provider

Reply:
Done